### PR TITLE
Devc/ops 871 add renovate mergify rules to contra

### PR DESCRIPTION
**Pull request by dev31000**
* Add mergify and renovate configs
* Add feature workflow

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      - label!=do-not-update
+      - '#approved-reviews-by>=1'
+    actions:
+      update:
+  - name: label PRs ready for review
+    conditions:
+      - base=main
+      - -draft
+      - label!=blocked
+      - check-success=Test (yarn lint:cspell)
```
Comment (leekesler) [inline .mergify.yml:20]: should all be `check-success=Lint`

File: .mergify.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          - ready-for-review
+  - name: queue approved PR (low priority)
+    conditions:
+      - base=main
+      - -draft
+      - '#approved-reviews-by>=1'
+      - '#changes-requested-reviews-by=0'
+      - label!=blocked
+      - label=dependencies
+      - label!=hotfix
```
Comment (leekesler) [inline .mergify.yml:37]: remove hotfix, not needed to be seen

**Review by leekesler — APPROVED** [review]

File: .mergify.yml (hunk 1 above)
Comment (dev31000) [reply .mergify.yml:20]: forgot this [](edited-with-Axolo)

File: .mergify.yml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  - name: queue approved PR (medium priority)
+    conditions:
+      - base=main
+      - -draft
+      - '#approved-reviews-by>=1'
+      - '#changes-requested-reviews-by=0'
+      - label!=blocked
+      - label!=dependencies
+      - label!=high-priority
+      - label!=hotfix
```
Comment (leekesler) [inline .mergify.yml:56]: same thing

**Review by leekesler — CHANGES_REQUESTED** [review]

File: .mergify.yml (hunk 1 above)
Comment (dev31000) [reply .mergify.yml]: Fixed

File: .mergify.yml (hunk 3 above)
Comment (dev31000) [reply .mergify.yml]: Fixed

File: .mergify.yml (hunk 2 above)
Comment (dev31000) [reply .mergify.yml]: Fixed

**Review by leekesler — APPROVED** [review]

**Pull request merged**